### PR TITLE
Simplify code to append custom JavaScript

### PR DESCRIPTION
**Pull request by Mogztter**
* [x]  Add a test to verify that the custom JavaScript element is appended before or after the content (depending on the settings)

### CODE REVIEW

File: spec/asciidocify-custom-script-spec.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+describe('asciidocify', function () {
+  describe('custom script', function () {
+    it('should append a custom script before the content', function (done) {
```
Comment (Mogztter) [inline spec/asciidocify-custom-script-spec.js]: I actually catch a 🐛 when writing this test! ⛑

**Comment by mojavelinux** [comment]
Is the idea that the before hook can be used to register extensions for Asciidoctor?

**Comment by Mogztter** [comment]
Yes exactly but I want to refine this feature because I think that nobody
knows how to use it :)

**Comment by mojavelinux** [comment]
:+1: 

I'm definitely anticipating a need for this as part of Antora integration.

**Pull request merged**